scripts/evaluate_release_tag.py

```diff
@@ -22,49 +20,66 @@ def create_argument_parser() -> argparse.ArgumentParser:
 
     return parser
 
+
 def is_plain_version(version: Version) -> bool:
     """Check whether a version is a X.x.x release with no alpha or rc markers"""
     return version.base_version == str(version)
 
 
-def git_existing_tag_versions() -> Set[Text]:
+def git_existing_tag_versions() -> List[Version]:
     """Return all existing tags in the local git repo."""
 
     stdout = check_output(["git", "tag"])
     tags = set(stdout.decode().split("\n"))
     versions = [Version(tag) for tag in tags if is_valid_version(tag)]
     return versions
 
-def git_plain_tag_versions(versions: List[Version]) -> Set[Text]:
+
+def git_plain_tag_versions(versions: List[Version]) -> List[Version]:
     """Return non-alpha/rc existing tags"""
     return [version for version in versions if is_plain_version(version)]
 
 
-def main(args: argparse.Namespace) -> None:
-    tag = Version(args.tag)
+def filter_non_alpha_releases(tags: List[Version]) -> List[Version]:
+    return [tag for tag in tags if tag.is_alpha is False]
```

Comment:
It's more pythonic to write it this way:
```suggestion
    return [tag for tag in tags if not tag.is_alpha]
```